src/encoded/schemas/changelogs/functional_characterization_experiment.md

```diff
@@ -1,5 +1,8 @@
 ## Changelog for functional_characterization_experiment.json
 
+### Schema version 10
+* The pooled clone sequencing assay term name require specification of the control_type
```

Comment:
This is a very minor edit, but I would say "The pooled clone sequencing assay term name requires* specification of the control_type"

Reply:
fixed